SAM_IoT_Central.md

```diff
@@ -26,53 +26,63 @@ The web UI lets you quickly connect devices, monitor device conditions, create r
 
     <img src=".//media/image41.png" style="width:5.in;height:3.18982in" alt="A screenshot of a cell phone Description automatically generated" />
 
-4. Modify `AzureIotPnPDps/Source Files/app.c` with your
-    wireless router’s SSID and password (keeping the surrounding quotation marks for each); for example
-
-    ```c
-    #define APP_CFG_MAIN_WLAN_SSID "MY_WIFI_AP_SSID"
-    #define APP_CFG_MAIN_WLAN_PSK  "MY_WIFI_AP_PSWD"
-    ```
-
-    > [!TIP]  
-    > You may also configure the WiFi SSID and password during run-time (i.e. dynamically when the board is set up to run the application) using the `wifi` command via the Command Line Interface (CLI); e.g.
-    ```bash
-    wifi <MY_WIFI_AP_SSID>,<MY_WIFI_AP_PSWD>,2
-    ```
-
-5. Verify the project properties are set correctly before building the
+4. Verify the project properties are set correctly before building the
    project
 
     - Connect the board to PC, then make sure `CURIOSITY` device shows up as a disk drive in a `File Explorer` window
 
-    - Right-click the project `AzureIotPnPDps` &gt; select `Properties` &gt; Verify
-    that all Configuration settings are at least the minimum versions as
-    shown in the below screenshot (and that your SAM-IoT board is
-    selected as the Connected Hardware Tool). If any changes were made in the project properties window, make sure to hit the `Apply` button before hitting `OK`
-
-        <img src=".//media/image26.png" style="width:5.88652in;height:2.68981in" alt="A screenshot of a social media post Description automatically generated" />
+    - Right-click the project `AzureIotPnPDps` &gt; select `Properties` &gt; select "SAMD21-IoT WG-SN" for `Connected Hardware Tool` &gt; select the latest `SAMD21_DFP` version &gt; select the latest XC32 version for `Compiler Toolchain`. If any changes were made in the project properties window, the `Apply` button should become enabled.  Make sure to hit the `Apply` button before hitting `OK`
```

Comment:
curious why not just 
```suggestion
    - Right-click the project `AzureIotPnPDps` > select `Properties` >; select "SAMD21-IoT WG-SN" for `Connected Hardware Tool` > select the latest `SAMD21_DFP` version > select the latest XC32 version for `Compiler Toolchain`. If any changes were made in the project properties window, the `Apply` button should become enabled.  Make sure to hit the `Apply` button before hitting `OK`
```

Reply:
also
```suggestion
    - Right-click the project `AzureIotPnPDps` &gt; select `Properties` &gt; select "SAMD21-IoT WG-SN" for `Connected Hardware Tool` &gt; select the latest `SAMD21_DFP` version &gt; select the latest XC32 version for `Compiler Toolchain`. If any changes were made in the project properties window, the `Apply` button should become enabled.  Make sure to hit the `Apply` button before hitting `OK`.
```

Reply:
In Markdown, quotes in a certain part of the text are indicated using a blockquote element.  so you need escape <>